src/Functions/currentProfiles.cpp

```diff
@@ -17,39 +17,46 @@ namespace
 {
     enum class Kind
     {
-        CURRENT_PROFILES,
-        ENABLED_PROFILES,
-        DEFAULT_PROFILES,
+        currentProfiles,
+        enabledProfiles,
+        defaultProfiles,
```

Comment:
[14. For the constants in an enum, use CamelCase with a capital letter. ALL_CAPS is also acceptable. If the enum is non-local, use an enum class.](https://clickhouse.com/docs/en/development/style#how-to-write-code)

Reply:
Let's break the rule...